#! /bin/sh
patch -p0 -f $* < $0
exit $?

Make --version also print the version number of Gpg4win so that we 
can see in bug reports what patches have been used.

(For version < 2.0.11 chnage the patch line from 1023 to 868.)

--- jnlib/argparse.c
+++ jnlib/argparse.c
@@ -1023,7 +1023,7 @@
   fputs (strusage (11), stdout);
   if ((s=strusage (12)))
     printf (" (%s)", s );
-  printf (" %s\n", strusage (13) );
+  printf (" %s %s\n", strusage (13), "(Gpg4win @PACKAGE_VERSION@)");
   /* Additional version lines. */
   for (i=20; i < 30; i++)
     if ((s=strusage (i)))

